site/layouts/shortcodes/orange-supra.html

```diff
@@ -0,0 +1,15 @@
+{{- /*
+  Usage: `orange-supra`
+*/ -}}
+
+{{- $input := .Inner -}}
+
+<nav class="navbar navbar-dark navbar-expand-md bg-dark supra">
```

Comment:
Haven't seen it before but [there was a `role="navigation"` in v4](https://boosted.orange.com/docs/4.6/components/navbar-orange/#supra-bar)

Reply:
The `<nav>` element will automatically communicate a section has a role of navigation.

Source : https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/navigation_role

Reply:
`supra` will always be expanded so `navbar-expand` is enough no need of `-md`